### ISSUE TITLE
As a CoatRack user, I would like to have a quick overview about the "health" of my gateways

### ISSUE THREAD

**Issue by gr-hovest-atb** [coatrack/coatrack#83]
As a CoatRack user, I would like to have a quick overview about the "health" of my gateways. The minimal functionality would be to see, for each of my gateways, the time of the last interaction between CoatRack and that gateway (e.g. metrics transmission from gateway to coatrack admin app). 

It would be helpful to have this information as additional column in the gateway list "time of last interaction" (and ideally also as a widget in the Dashboard).

**Comment by BrunoSilvaATB** [comment]
![Gateway Health Monitor Frontend](https://user-images.githubusercontent.com/47450617/109507985-a0237880-7a9f-11eb-99c7-ee462fd9ff62.png)

**Comment by BrunoSilvaATB** [comment]
**Frontend** (@ChristophBaierATB)
- Gateway Health Monitor based on a picture
- Create the Info Dialog explaining the meaning of the states and usage of the Gateway Health Monitor 
- Create an Extra Column between Description and Download which shows the last gateway call time

**Backend** (@BrunoSilvaATB) 
- Add an attribute lastCallTimeToAdmin to the Gateway Class 
- For the Gateway Health States (Create an enumeration type which contains the states Ok, Warning, and Critical) 
- Add sample data

**Comment by BrunoSilvaATB** [comment]
Regarding the Gateway Health States, as soon as the admin is not getting a call from the gateway in 5 min, then the user should see a warning on that specific gateway. For example, an orange color

**Comment by BrunoSilvaATB** [comment]
Regarding the connection between Gateway and Admin, only minutes should be displayed. so the exact number would be 5 minutes, even if the call took 5m40s. However, as longs as this call take more than 6 minutes then a warning should be displayed for this gateway.

Regarding pagination, as on the moment there are not so many Gateways then the pagination is low priority.

**Comment by Monezz** [comment]
I think we should leverage the spring infrastructure to collect this data.
Spring supports for example micrometer, which can collect a range of metrics we can use to build gui components on.
https://spring.io/blog/2018/03/16/micrometer-spring-boot-2-s-new-application-metrics-collector

If we use it in combination with a system that can collect the data in the admin in a MeterRegistry, it would save a lot of time.

**Comment by BrunoSilvaATB** [comment]
https://micrometer.io/

**Comment by gr-hovest-atb** [comment]
The feature we started to discuss about in this issue is very simple to implement: after #73 will be done, the gateway will anyway call admin every few minutes to get an updated api key list. We just need to store a timestamp on each of that calls and would have enough information to create this simple health monitor as sketched above.

A more elaborate version including flexible different metrics, based on micrometer etc. should be discussed in a separate issue.
